presto-main/src/main/java/com/facebook/presto/sql/gen/CompilerOperations.java

```diff
@@ -64,6 +65,11 @@ public static boolean in(Object value, Set<?> set)
         return set.contains(value);
     }
 
+    public static void setBytes(int index, Slice source, Slice destination)
```

Comment:
Not needed with `ByteCodeExpression` (which is awesome! try to use it everywhere you can)